src/main/java/org/mule/extension/http/internal/listener/HttpListener.java

```diff
@@ -201,7 +205,9 @@ public void onError(@ParameterGroup(name = "Error Response",
                       Error error,
                       SourceCompletionCallback completionCallback) {
     try {
-      sendErrorResponse(errorResponse, callbackContext, error, completionCallback);
+      if (!error.getCause().getClass().getName().equals(CONNECTION_ALREADY_CLOSED_EXCEPTION)) {
```

Comment:
from what i talked yesterday with @mlischetti , we may need a more generic approach in the runtime to handle this cases, otherwise this same situation can happen in other connectors trying to send data to closed connections